cfme/containers/provider/__init__.py

```diff
@@ -355,6 +361,45 @@ def step(self):
         sel.click(InfoBlock('Overview', 'Topology'))
 
 
+class AdHocMetricsView(BaseLoggedInPage):
+    filter_dropdown = SelectorDropdown('uib-tooltip', 'Filter by')
+    apply_btn = form_buttons.apply_filters
```

Comment:
@shalomnaim1 if you want to play with it from the console you should something like this:
```python
from utils.appliance import get_or_create_current_appliance
appliance = get_or_create_current_appliance()
from widgetastic_patternfly import Button
apply_btn = Button("Apply Filters")
apply_btn.click()
```
Consider to read widgetastic documentation https://mojo.redhat.com/docs/DOC-1125600